lib/order.rb

```diff
@@ -0,0 +1,32 @@
+require './lib/menu.rb'
+
+class Order
+
+attr_reader :current_order, :current_menu, :item_no, :item_total
+
+    def initialize(menu_klass: Menu)
+      @current_order = []
+      @current_menu = []
+      @menu_klass = menu_klass
+    end
+
+    def start_order(item, item_no)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for start_order is too high. [24/15] (http://c2.com/cgi/wiki?AbcMetric)